### PR TITLE
update identity path for consent

### PR DESCRIPTION
**Pull request by stuart-bass-bjss**

### CODE REVIEW

File: environments/azure.yaml [hunk 1]
```diff
@@ -83,7 +83,7 @@ isapi:
           - /identity/css/*  
           - /identity/nhsuk-frontend/*          
           - /identity/account/*          
-          - /identity/consent/*          
+          - /identity/consent*          
           - /identity/error/*
           - /identity/connect/*
           - /identity/.well-known/*
```
Comment (assoomro) [inline environments/azure.yaml:89]: Should we change the paths to match the pattern in all other instances?

Comment (stuart-bass-bjss) [reply environments/azure.yaml:89]: that was the one I *had* to change

Comment (assoomro) [reply environments/azure.yaml:89]: Are the other ones going ok through the ingress then?

Comment (stuart-bass-bjss) [reply environments/azure.yaml:89]: they seem to be. I wanted to minimise change at this point.

Comment (assoomro) [reply environments/azure.yaml:89]: 👍 Just wanted to check

**Pull request merged**